src/App.jsx

```diff
@@ -1,107 +1,30 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
+
+function getPostById(id) {
```

Comment:
Wrong function name, we are getting comments here
```suggestion
function getCommentsByPostId(postId) {
```